modules/govuk/manifests/apps/email_alert_api/sidekiq_queue_check.pp

```diff
@@ -0,0 +1,38 @@
+# == Define: govuk::apps::email_alert_api::sidekiq_queue_checks
+#
+# Creates an Icinga check which checks Graphite for activity
+# on Sidekiq queues
+#
+# === Parameters
+#
+# [*ensure*]
+#   Whether to enable the check.
+#   Default: present
+#
+# [*title*]
+#   The name of the sidekiq queue to check.
+#
+# [*size_warning*]
+#   The number of messages on a queue to trigger a warning.
+#
+# [*size_critical*]
+#   The number of messages on a queue to trigger a critical alert.
+#
+define govuk::apps::email_alert_api::sidekiq_queue_check(
+  $ensure = present,
```

Comment:
You probably don't need `$ensure` as a parameter.

It's useful when you're explicitly managing resources, like setting them to `$ensure => absent`, but with the one resource being managed here, you don't need to do that, as Puppet manages all the Icinga configuration, including removing old config [1].

1: https://github.com/alphagov/govuk-puppet/blob/master/modules/icinga/manifests/config.pp#L45-L47You probably don't need `$ensure` as a parameter.

It's useful when you're explicitly managing resources, like setting them to `$ensure => absent`, but with the one resource being managed here, you don't need to do that, as Puppet manages all the Icinga configuration, including removing old config [1].

1: https://github.com/alphagov/govuk-puppet/blob/master/modules/icinga/manifests/config.pp#L45-L47